vignettes/intro.Rmd

```diff
@@ -125,3 +125,36 @@ robin_glm(y_b ~ treatment * s1 + covar,
   family = binomial(link = "logit")
 )
 ```
+
+## Obtain the Confidence Intervals for the Marginal Means and contrast
+
+To obtain the confidence interval, we can use `confint` function.
+
+Given the following model
+```{r}
+robin_res <- robin_glm(y_b ~ treatment * s1 + covar,
+  data = glm_data,
+  treatment = treatment ~ ps(s1),
+  family = binomial(link = "logit"),
+  contrast = "log_risk_ratio"
+)
+robin_res
+```
+
+It is easy to obtain the confidence interval matrix for the marginal mean, at specified level.
+If `parm` is not provided, the complete matrix (all treatment groups) will be provided.
+
+```{r}
+confint(robin_res$marginal_mean, parm = 1:2, level = 0.7)
```

Comment:
would be nice to add the `marginal_mean()` and `contrast()` accessor generic functions/methods as discussed on Monday